views/favorites.ejs

```diff
@@ -9,20 +9,20 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Favorites</title>
+    <title>Favorite Games</title>
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-    <h1>Profile Page</h1>
+    <h1>User's Game List</h1>
     <ul>
     <% faves.forEach( el => { %> 
             <li class='faveGameItem' data-id='<%=el._id%>'>
-         
+                <span class= "Game" class="<%= el.boardGame === true ? 'Board Game' : el.cardGame === true ? 'Card Game' : 'Video Game'%>"<%= el.faves %></span>
```

Comment:
@allistersascha 
I think that you will want to make the class added one word, otherwise the space would be interpreted as adding two separate classes, "Board" and "Game"

So el.boardGame === true ? 'boardGame'

Reply:
Gotcha, added this plus <%=el.Game%>, will update with that soon